### DISCUSSION TITLE
Add the bitrate option and lens mode for the live-streaming QR code.

### DISCUSSION THREAD

**Discussion by rickcam9000** [gopro/labs#275] [Feature Requests]
You can set the bitrate to a higher value and choose a different lens when launching the livestream via bluetooth.
But using the QR code you only end up with the defaults

**Comment by WURS5000** [comment]
In general i'd like to see a feature where we can set metadata for live-streaming.

Something like BITR for livestreaming would be great

**Comment by ysliew** [comment]
Wouldn't it be better to allow setting of arbitrary video and audio bit rate via QR for live-streaming?

In some cases, we value quality over speed which high bitrates is desirable.

In other cases when the WIFI of 4G mobile internet signal is  simply bad/slow, low bitrate (i.e. 2 Mbps) at least give us some stable  and 'smooth'  feed by compromising the video quality.

**Reply by dnewman-gpsw** [reply]
This feature was added to HERO11 and 12.  oMBITL=5 sets the upper bit-rate for livestreaming.
